baram/ec2_manager.py

```diff
@@ -299,6 +299,9 @@ def list_detail_subnets(self) -> list:
                  'state': subnet['State']})
         return subnet_list
 
+    def get_subnet_ids_with_vpc_id(self, vpc_id: str):
```

Comment:
please add return type hint and description comment.

Reply:
sure I will add them